### PR TITLE
Fix [SplitV CPU Kernel] condition error for parallelism output judgement.

### PR DESCRIPTION
**Pull request by kitstar**
The condition wants to make sure each intra-thread has enough work.
The detail requirement should be:
if num_splits < num_threads, then we just use **num_splits** intra-threads to work, each intra-thread should handle at least 4096 elements.
if num_splits >= num_threads, then we use **num_threads** intra-threads to work, each intra-thread should handle multi-splits, avg sigma( elements of num_spilits / num_threads) > 4096.

So the condition should be **min**(num_splits, num_threads), not **max**.

### CODE REVIEW

**Comment by gbaned** [comment]
@kitstar Thank you for your contribution. Can you please sign CLA? Thanks!

**Comment by kitstar** [comment]
@googlebot I fixed it.

**Comment by kitstar** [comment]
@googlebot I fixed it.

File: tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc [hunk 1]
```diff
@@ -408,7 +408,7 @@ Status CreateTRTNode(const ConversionParams& params,
           }
         }
 
-        // Rewrire data input if it's not found in original graph.
```
Comment (bixia1) [inline tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc:411]: Thank you for catching this typo! Would you please split this to another PR, to avoid mixing changes to two different components into one PR?

**Comment by bixia1** [comment]
We may want to ask @jewillco to review the change to split_v_op.cc.

File: tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc (hunk 1 above)
Comment (jewillco) [reply tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc:411]: Are you sure this wasn't meant to be "rewire"?

Comment (bixia1) [reply tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc:411]: It is "rewire", it was "rewrire".

Comment (kitstar) [reply tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc:411]: Fixed.

Comment (bixia1) [reply tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc:411]: Would you please split this change to another PR? There are reasons for this: (1) the PR description doesn't mentioned this change. (2) changes to tf2tensorrt has a more complicated process for merging and can't be merged automatically.  (3) allow "owners" to the two components to approve changes for each component.

**Comment by gbaned** [comment]
@jewillco Can you please take a look on this PR ? Thanks!

File: tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc (hunk 1 above)
Comment (kitstar) [reply tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc]: Sure. Typo fix is reverted.

**Comment by jewillco** [comment]
I am fine with the CL but I'm not allowed to merge it myself.

**Review by bixia1 — APPROVED** [review]
